<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 170, 255)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit_source_english_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>330</width>
      <height>350</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(0, 0, 0);
	border-radius: 12px;
	font: 14pt &quot;Sans Serif Collection&quot;;
}</string>
    </property>
    <property name="placeholderText">
     <string>Enter text to be translated</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_output">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>330</width>
      <height>350</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit {
	background-color: rgb(255, 255, 255);
	border: 2px solid rgb(0, 0, 0);
	border-radius: 12px;
	font: 14pt &quot;Sans Serif Collection&quot;;
}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Your translation will be displayed here</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>240</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif Collection</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pig Latin Translator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_translate">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>480</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans Serif Collection</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(255, 0, 255);
	border: 2px solid rgb(0, 0, 0);
	border-radius: 12px;
}

QPushButton::hover {
	background-color: rgb(255, 85, 255);
}

QPushButton::pressed {
	background-color: rgb(230, 0, 255);
}</string>
    </property>
    <property name="text">
     <string>Translate</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="sadPig">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>480</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../assets/peppaPigSad.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="happyPig">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>470</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../assets/peppaPigHappy.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>textEdit_source_english_input</sender>
   <signal>textChanged()</signal>
   <receiver>textEdit_output</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_translate</sender>
   <signal>pressed()</signal>
   <receiver>textEdit_output</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>428</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_translate</sender>
   <signal>pressed()</signal>
   <receiver>textEdit_output</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>619</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>textEdit_source_english_input</sender>
   <signal>textChanged()</signal>
   <receiver>sadPig</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>506</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_translate</sender>
   <signal>pressed()</signal>
   <receiver>sadPig</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>530</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
